sources/clinicaltrialsgov.py

```diff
@@ -3,10 +3,30 @@
 import utils
 from dateutil import parser
 from datetime import timezone
+from bs4 import BeautifulSoup
 
 FILENAME = "clinicaltrialsgov.json"
 POSTED_WITHIN_DAYS = 200  # posted within the last X days
 
+
+def summary_to_dict(summary):
+    fields = {}
+    soup = BeautifulSoup(summary, "html.parser")
+    for b in soup.findAll("b"):
+        key = b.get_text()
+        # remove colon
+        value = b.next_sibling and b.next_sibling[1:].replace("\xa0", " ").strip()
+        # if 'recruit' in key.lower() and not value:
+        #     value = key
+        #     key = 'Stage'
+        fields[key] = value
```

Comment:
the b.text() keys won't be standardized across all registries. can we hardcode some of them?

Reply:
Yep